src/Entity/Competition/Competition.php

```diff
@@ -0,0 +1,103 @@
+<?php
+
+namespace BalticRobo\Api\Entity\Competition;
+
+use BalticRobo\Api\Model\Cms\AddCompetitionDTO;
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * @ORM\Entity(repositoryClass="BalticRobo\Api\Repository\Competition\CompetitionRepository")
```

Comment:
You can remove `repositoryClass`. We use our repositories as services so it won't be used anymore.

You forgot to add `ORM\Table()` where you add name of table in database. Remember that table name HAVE TO by plural.